src/coffee/services/s3client.coffee

```diff
@@ -220,10 +220,14 @@ class S3Client
           Compress.compressImage(tmp_resized, tmpDir, extension)
       .then =>
         @sendMetrics 'increment', 'image.resized'
-        header = 'x-amz-acl': 'public-read'
+        if isNaN expire
+          expire = 2592000
```

Comment:
No I mean to set the number here. If it's a default, you should set it in the command option itself. And also what does this number mean? I guess it's 30days, but add a comment next to it with the days value.

PS: please don't mention customer names, it's a public repo!